doc/getting_started/beginner_tutorial.rst

```diff
@@ -25,163 +22,245 @@ Create a Repo
 A ``repo`` is the highest level primitive in the Pachyderm file system (pfs). Like all primitives in pfs, it shares it's name with a primitive in Git and is designed to behave analogously. Generally, repos should be dedicated to a single source of data such as log messages from a particular service, a users table, or training data for an ML model. Repos are dirt cheap so don't be shy about making tons of them.
```

Comment:
Do we want  talk a lot about PFS here?  Maybe "highest level data primitive in Pachyderm.  Like many things in Pachyderm, it shares it's name with primitives of Git..."